### PR TITLE
Add admin reg blog post

### PR DESCRIPTION
**Pull request by mooreds**
This is a post about the admin user and registration edit screens released in 1.20

### CODE REVIEW

**Comment by robotdan** [comment]
I like it!

File: site/_posts/2020-11-06-custom-admin-registration-forms.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+layout: blog-post
+title: Update user profile data in the admin UI
```
Comment (dpolstra) [inline site/_posts/2020-11-06-custom-admin-registration-forms.md:3]: How to update user profile data in the FusionAuth admin UI

Comment (dpolstra) [reply site/_posts/2020-11-06-custom-admin-registration-forms.md:3]: Or "Using advanced registration forms to update user profile data"

Comment (mooreds) [reply site/_posts/2020-11-06-custom-admin-registration-forms.md:3]: Hmm, you're right that it doesn't pop.

What we are basically doing is letting FusionAuth admin users who build custom user profile editing forms.

How about 

* Build custom user profile edit forms using FusionAuth
* How to build custom profile edit forms within FusionAuth
* Manage custom user profile data in the FusionAuth admin

**Pull request merged**